### ISSUE TITLE
Docs: Search API sort=score

### ISSUE THREAD

**Issue by g-saracca** [IQSS/dataverse#11105]
It would be nice if the [Search API documentation](https://guides.dataverse.org/en/latest/api/search.html#parameters) would mention the extra value for the sort parameter which is `score` and how the response behaves based on this.

**Comment by pdurbin** [comment]
Some more context [from Slack](https://iqss.slack.com/archives/C04MJ76A34Z/p1734449253231269):

![Screenshot 2024-12-17 at 11 21 57 AM](https://github.com/user-attachments/assets/0204cb98-2662-43eb-9bfd-19181624a161)

Also, here's how "Relevance" looks in the UI after you do a search:

![Screenshot 2024-12-17 at 11 22 40 AM](https://github.com/user-attachments/assets/fc1c052c-94a6-451b-ac5d-99d63d34ccdf)

Here's the file to edit: https://github.com/IQSS/dataverse/blob/develop/doc/sphinx-guides/source/api/search.rst

Maybe I'll see if someone from the community would like to work on this. 🤔

**Comment by shikhar1est** [comment]
Hi @pdurbin , is this issue still open to work? If it is I can work on this and solve the issue.

**Comment by pdurbin** [comment]
@shikhar1est yes! Please go ahead! If you have any questions please ask here or at https://dataverse.zulipchat.com/#narrow/channel/446770-docs/topic/add.20.22score.22.20to.20Search.20API.20docs/near/489525150

For how to work on the docs, there's an updated version at https://dataverse-guide--11831.org.readthedocs.build/en/11831/contributor/documentation.html (not merged yet, from #11831).

**Comment by pdurbin** [comment]
@shikhar1est I'm going to let @superpegaso2703 take a swing at this.
